stock_request_picking_type/tests/test_stock_request.py

```diff
@@ -34,47 +34,35 @@ def setUpClass(cls):
             .with_context(company_id=cls.company_2.id)
             .create({"name": "Peñiscola"})
         )
-        cls.product = cls._create_product("SH", "Shoes", False)
-        cls.product_company_2 = cls._create_product("SH_2", "Shoes", cls.company_2.id)
-        cls.stock_request_user = cls._create_user(
-            "stock_request_user",
-            [cls.stock_request_user_group.id],
-            [cls.main_company.id, cls.company_2.id],
+
+        cls.product = cls.env["product.product"].create(
+            {"default_code": "SH", "name": "Shoes", "company_id": False}
+        )
+
+        cls.product_company_2 = cls.env["product.product"].create(
+            {"default_code": "SH_2", "name": "Shoes", "company_id": cls.company_2.id}
         )
 
-    def _create_user(self, name, group_ids, company_ids):
-        return (
-            self.env["res.users"]
+        cls.stock_request_user = (
+            cls.env["res.users"]
             .with_context(no_reset_password=True)
             .create(
                 {
-                    "name": name,
+                    "name": "stock_request_user",
                     "password": "demo",
-                    "login": name,
-                    "email": "@".join([name, "test.com"]),
-                    "groups_id": [(6, 0, group_ids)],
-                    "company_ids": [(6, 0, company_ids)],
+                    "login": "stock_request_user",
+                    "email": "@".join(["stock_request_user", "test.com"]),
+                    "groups_id": [(6, 0, [cls.stock_request_user_group.id])],
+                    "company_ids": [(6, 0, [cls.main_company.id, cls.company_2.id])],
                 }
             )
         )
 
-    def _create_product(self, default_code, name, company_id, **vals):
```

Comment:
Don't remove those kind of helper methods.

Reply:
Done!